util/metrics/metrics.go

```diff
@@ -0,0 +1,20 @@
+package metrics
+
+import (
+	"fmt"
+	"regexp"
+)
+
+// Prometheus invalid labels, more info: https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels.
+var invalidPromLabelChars = regexp.MustCompile(`[^a-zA-Z0-9_]`)
+
+func NormalizeLabels(prefix string, labels []string) []string {
```

Comment:
because you already moved it, can you cover this util with tests ?

Reply:
Add unit test